sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -713,6 +722,16 @@ public Read withTemplateCompatibility() {
       DIRECT_READ,
     }
 
+    /** Determines the data format used when reading/writing in {@link Method#DIRECT_READ} mode. */
+    @Experimental(Experimental.Kind.SOURCE_SINK)
+    public enum DataFormat {
+      /** Utilize Avro data format. */
```

Comment:
I think it would make sense to have a `DEFAULT` here like we do in `Method`. Then we can switch the default over to arrow when its stable.

Reply:
Makes sense, having a default option will make in consistent with previous work (`Method`).